### PR TITLE
GS: fix keyboard brightness icons

### PR DESCRIPTION
**Pull request by Jupi007**
- centre keyboard brightness icons ;
- add missing `keyboard-brightness-off-symbolic` icon.

Capture d’écran du 2023-08-23 17-07-12

Replace #3969
Related to #3966

### CODE REVIEW

**Comment by 3v1n0** [comment]
Looks nice, I kinda of liked the padding between the "off" slash and the keystroke of upstream version, is something we should keep?

File: gnome-shell/src/data/gnome-shell-icons.gresource.xml [hunk 1]
```diff
@@ -25,6 +25,7 @@
     <file>scalable/status/keyboard-brightness-high-symbolic.svg</file>
     <file>scalable/status/keyboard-brightness-low-symbolic.svg</file>
```
Comment (3v1n0) [inline gnome-shell/src/data/gnome-shell-icons.gresource.xml:26]: This can be removed from gresources as per #3966

**Comment by Jupi007** [comment]
Well I added one. We also do this in Yaru icons, the removed space is up to the slash.
But on this icon, there was only a very small part removed.

**Comment by Jupi007** [comment]
I updated the "off" icon to be consistent with our other "off" icon (like sound volume):

image

**Comment by 3v1n0** [comment]
I.e. something like this is too much?

image

**Comment by Jupi007** [comment]
Well the erased part is always up to the slash:

image

So the maximum I can do is this:

image

**Comment by 3v1n0** [comment]
Ack, that's better.

**Comment by 3v1n0** [comment]
Thanks!

**Pull request merged**